rust/docker-compose.yaml

```diff
@@ -1,4 +1,4 @@
-version: "3"
+version: "3.4"
```

Comment:
This is a downgrade in version 3 == latest version in 3 which is currently 3.9
```suggestion
version: "3"
```

Reply:
Okey! Its just that It was giving an error when I tried to build the containers, maybe I need to update my docker

Reply:
`docker-compose build
ERROR: The Compose file './docker-compose.yaml' is invalid because:
services.rust-tests.build contains unsupported option: 'cache_from'
services.rust-tests.volumes contains an invalid type, it should be a string
services.rust-tests.volumes contains an invalid type, it should be a string
services.rust-tests.volumes contains an invalid type, it should be a string
services.rust-tests.volumes contains an invalid type, it should be a string`

Reply:
Yep, I've updated docker compose and now it works! :p